modules/core/src/test/java/org/apache/ignite/testframework/junits/IgniteTestResources.java

```diff
@@ -93,19 +97,22 @@ public IgniteTestResources() throws IgniteCheckedException {
         else
             log = rootLog.getLogger(getClass());
 
-        this.jmx = prepareMBeanServer();
+        jmx = prepareMBeanServer();
+
+        ctx = new GridTestKernalContext(log);
 
-        this.rsrcProc = new GridResourceProcessor(new GridTestKernalContext(this.log));
+        rsrcProc = new GridResourceProcessor(ctx);
     }
 
     /**
      * @param cfg Ignite configuration
      */
     public IgniteTestResources(IgniteConfiguration cfg) throws IgniteCheckedException {
         this.cfg = cfg;
-        this.log = rootLog.getLogger(getClass());
-        this.jmx = prepareMBeanServer();
-        this.rsrcProc = new GridResourceProcessor(new GridTestKernalContext(this.log, this.cfg));
+        log = rootLog.getLogger(getClass());
```

Comment:
why this change?

Reply:
"this." was not necessary so I removed it while adding new field.